lms/templates/ccx/underscore/form.underscore

```diff
@@ -0,0 +1,181 @@
+<div class="field">
```

Comment:
XSS best practices need to be followed-- there are unescaped strings in here like "chapter.display_name" and "date" (in template above). All strings should be escaped unless they include HTML. If they include HTML, they must be properly wrapped using HTMLUtils methods.

Please read over the new XSS best practices document-- http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/conventions/safe_templates.html